src/docs/data.json

```
@@ -21,6 +21,9 @@
     "Reference": [
       "Configuration",
       "Permissions Disclosure"
+    ],
+    "Questions": [
+      "Questions"
```

I feel this is a bit vague. perhaps we can go with something like the below. Later, a FAQ page would also fit in this generic "Support" category

```suggestion
    "Support": [
      "Ask Us a Question"
```